modules/op-scoping-github-token.adoc

```diff
@@ -0,0 +1,103 @@
+
+By default, this token is scoped only to the repository where the payload comes from.
+However, in some cases, the developer team might want the token to allow control over additional repositories. For example, there might be a CI repository where the `.tekton/pr.yaml` file and source payload might be located, however the build process defined in `pr.yaml` might fetch tasks from a separate private CD repository.
+
+You can extend the scope of the GitHub token in two ways:
+
+* _Global configuration_: extend the GitHub token to a list of repositories in different namespaces. You must have administrative permissions to set this configuration.
+* _Repository level configuration_: extend the GitHub token to a list of repositories that exist in the same namespace as the original repository. You do not require administrative permissions to set this configuration.
+
+.Procedure
+
+. In , set the `secret-github-app-token-scoped` key to `false`. This setting enables the scoping of the GitHub token to private and public repositories listed under the Global and Repository level configuration.
+
+. To set global configuration for scoping the GitHub token, in the `TektonConfig` custom resource, under `pipelinesAsCode` and `settings`, specify the additional repositories the `secret-github-app-scope-extra-repos` key, as in the following example:
++
+[source,yaml]
+----
+  spec:
+      platforms:
+        openshift:
+          pipelinesAsCode:
+            enable: true
+            settings:
+              secret-github-app-token-scoped: false
+              secret-github-app-scope-extra-repos: "owner2/project2, owner3/project3"
+----
++
+. To set repository level configuration for scoping the GitHub token, specify the additional repositories the `github_app_token_scope_repos` key, as in the following example:
++
+[source,yaml]
+----
+  apiVersion: "pipelinesascode.tekton.dev/v1alpha1"
+  kind: Repository
+  metadata:
+    name: test
+    namespace: test-repo
+  spec:
+    url: "https://github.com/linda/project"
+    settings:
+      github_app_token_scope_repos:
+      - "owner/project"
+      - "owner1/project1"
+----
++
+In this example, the `Repository` custom resource is associated with the `linda/project` repository in the `test-repo` namespace. The scope of the generated GitHub token is extended to the `owner/project` and `owner1/project1` repositories, as well as the `linda/project` repository. These repositories must exist under the `test-repo` namespace.
++
+[NOTE]
+====
+The additional repositories can be public or private, but must reside in the same namespace as the repository with which the `Repository` resource is associated.
+
+If any of the repositories does not exist in the namespace, the scoping of the GitHub token fails with an error message as in the following example:
+
+[,console]
```

Comment:
I haven't seen `[,console]` used before; I think we generally go with `[source,terminal]`, e.g. https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output